Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -326,6 +327,29 @@ protected override void Dispose(bool disposing)
 					List = null;
 				}
 
+				if (Element != null)
```

Comment:
how do the other platforms handle this? Should we do this on Android/iOS too?

Reply:
Nope. On Android, the `ListViewAdapter` keeps track of **all** created views in its field
```readonly List<ConditionalFocusLayout> _layoutsCreated = new List<ConditionalFocusLayout>();```

I did not check iOS, but I guess this works as well - I do not have the necessary tooling to check this anyways.

This seems to be a UWP bug only